src/pages/IdeaDetail.jsx

```diff
@@ -1,10 +1,27 @@
+import { useState, useEffect } from "react";
 import { useParams } from "react-router-dom";
 import useWorkshop from "../hooks/use-workshop";
 import Button from "../components/Button";
 
 export default function LoginPage() {
```

Comment:
Please change the name to IdeaDetail

Reply:
On it! I fixed it in one version, then forgot to update it